src/lib/src/common/workspace.ts

```diff
@@ -76,6 +84,34 @@ export class Workspace {
     }
   }
 
+  observeMarker<VALUE_TYPE>(observer: MarkerObserver<VALUE_TYPE>): void {
+    if (this.pathToElement.get(observer.path)) {
+      if (!this.hasMarker(observer.path, observer.field)) {
+        this.setMarkerValue(observer.path, observer.field, null);
+      }
+      observer.observe().then((value) => this.setObservedMarker(observer, value))
+      .catch((reason) => {
+        console.log(reason);
+        this.setObservedMarker(observer, this.getMarkerValue(observer.path, observer.field));
```

Comment:
is this another start of the polling? maybe add a comment here that this set is necessary to start the observer again, or log this information additionally.

Reply:
Yes, I want the polling to continue even if the server does not respond without error every time.
This can and should be improved, though: First, you'd probably want a waiting period before retrying. Even for successful requests, such a minimum waiting time would be sensible, as right now, this implementation relies on the server to implement a waiting period (i.e. it should hold the request open for a while, unless a state change occurs during that period, as we do for test state updates). If the server always responds as fast as it can, it could get flooded in requests… Secondly, after a number of unsuccessful requests, one might want to give up polling as the server or the connection will likely be down.

Reply:
Had added a comment, but since this logic was duplicated anyways, I introduced a private method whose name `logObserverErrorAndContinue` hopefully speaks for itself.